examples/verifying_fd_afd.py

```diff
@@ -0,0 +1,63 @@
+import desbordante
+import pandas as pd
+
+
+TABLE = 'examples/datasets/DnD.csv'
+COLOR_CODES = {'green': '\033[1;42m', 'red': '\033[1;41m', 'default': '\033[1;49m'}
```

Comment:
Should be constants in the code

Reply:
I didn't understand it a bit. It's a python, there are no constants, aren't there? Or is it meant to put it somewhere in a separate file or something?

Reply:
There are no actual constants, yes. I meant that those color codes should be their own variables. See also [here](https://peps.python.org/pep-0008/#constants)